### PR TITLE
Added explaination for Taints and Tolerations

### PR DESCRIPTION
**Pull request by tamilselvan1102**
Added explaination for Taints and Tolerations
Link #39442

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ weight: 50
 
 
 <!-- overview -->
-[_Node affinity_](/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
-is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attracts* them to
+Taints are a Kubernetes node property that enable nodes to repel certain pods. Tolerations are a Kubernetes pod property that allow a pod to be scheduled on a node with a matching taint.
+{{< glossary_tooltip text="Node affinity" term_id="node-affinity" >}} is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attracts* them to
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
-hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
+hard requirement). _Taints_ are the opposite.Taints allow a node to repel a set of pods.
```
Comment (NitishKumar06) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:16]: why this change?

**Review by NitishKumar06 — COMMENTED** [review]
Resolve the failing checks as well.

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md (hunk 1 above)
Comment (tamilselvan1102) [reply content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:16]: > why this change?

suggested in the issue 39442

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ weight: 50
 
 
 <!-- overview -->
+Taints are a Kubernetes node property that enable nodes to repel certain pods. Tolerations are a Kubernetes pod property that allow a pod to be scheduled on a node with a matching taint.
+
 [_Node affinity_](/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
 is a property of {{< glossary_tooltip text="Pods" term_id="pod" >}} that *attracts* them to
 a set of {{< glossary_tooltip text="nodes" term_id="node" >}} (either as a preference or a
-hard requirement). _Taints_ are the opposite -- they allow a node to repel a set of pods.
+hard requirement). _Taints_ are the opposite and allow a node to repel a set of pods.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:18]: This change isn't required and the text is correct as-is.

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 3]
```diff
@@ -10,13 +10,15 @@ weight: 50
...
 
 _Tolerations_ are applied to pods. Tolerations allow the scheduler to schedule pods with matching
-taints. Tolerations allow scheduling but don't guarantee scheduling: the scheduler also
+taints. Tolerations allow scheduling but don't guarantee scheduling. The scheduler also
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:21]: This change isn't required and the text is correct as-is.

**Review by divya-mohan0209 — COMMENTED** [review]
Please make the necessary amendments @tamilselvan1102 & we'll proceed for tech review + merge.

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md (hunk 3 above)
Comment (tamilselvan1102) [reply content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:21]: @divya-mohan0209 
pointed changed has been reverted as per your instruction. 
Please confirm other changes.

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ weight: 50
 
 
 <!-- overview -->
+Taints are a Kubernetes node property that enable nodes to repel certain pods. Tolerations are a Kubernetes pod property that allow a pod to be scheduled on a node with a matching taint.
+
```
Comment (nitishfy) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:14]: maybe we can also mention that _Taints and Toleratioins_ doesn't guarantee restriction of other Pod to a node in which taint has been applied.

**Comment by kbhawkey** [comment]
Hello @tamilselvan1102.
I plan to close this pull request. If you have questions or comments, feel free to reopen the PR.
Thanks
/close
